src/pages/getting-started/index.mdx

```diff
@@ -35,7 +35,7 @@ $ ./docker/up.sh
 ```
 
 :::info
-Pass the `--build` flag to the script to build images from source, or `--tag X.Y.Z` to use a tagged image.
+Pass the `--build` flag to the script to build images from source, or `--tag X.Y.Z` to use a tagged image. Pass `--seed` to seed Marquez with dummy metadata. Be advised that the lineage graph produced by this quickstart will differ as a result. 
```

Comment:
I've opened a fix for the incorrect rendering of the lineage graph https://github.com/MarquezProject/marquez/pull/2519. So, we can remove the comment on the `--seed` usage.